src/styles.css

```diff
@@ -1,125 +1,134 @@
 * {
```

Comment:
We need to revisit this CSS file as it appears something has happened with our indentation!